lib/ability.rb

```diff
@@ -19,6 +19,7 @@ def initialize(user, resource, controller = nil)
         can :manage, UserPriceGroupMember if resource.admin_editable?
         can :manage, AccountPriceGroupMember
       else
+        can([:edit, :update], User) { |target_user| target_user.external? }
```

Comment:
That's what I did originally, and it wasn't working :( but I will add some tests at least, thanks!